src/oc_erchef/apps/chef_objects/test/chef_s3_tests.erl

```diff
@@ -60,16 +60,16 @@ port_or_no_port(InternalS3Url, ExternalS3Url) ->
 
 % construct url from aws_config record
 get_host_test() ->
-    Config0 = mini_s3:new("", "", "http://1.2.3.4" ),
+    Config0 = mini_s3:new("", "", "http://1.2.3.4",     path),
```

Comment:
Is there value in tests here to use  `vhost`?

Reply:
I actually tried that (originally), and had code for it.

1) It didn't work.  Some kind of environmental issue with bringing in the config.  Ran out of time fooling with it if we want to do a release.

2) vhost will only work against S3, which we don't test against now (we only test against bookshelf).  So to be consistent with what we are already doing (just testing against bookshelf), I removed all of that code, and assumed testing against S3 is a "one off" kind of thing.

Having said that, there may be value in it.  Would it be best to consider this as an issue for future work?  Do you want me to write it up?

Reply:
Yeah, let's plan that for post-release.